backend/src/main/java/fk/prof/backend/util/PathNamingUtil.java

```diff
@@ -21,7 +26,11 @@ public static String encode32(String str) {
 
     }
 
-    public static String getDirectoryPath(Recorder.ProcessGroup processGroup) {
-        return DELIMITER + encode32(processGroup.getAppId()) + DELIMITER + encode32(processGroup.getCluster()) + DELIMITER + encode32(processGroup.getProcName());
+    public static String getPolicyNodePath(Recorder.ProcessGroup processGroup, String policyRootPath, String policyVersion) {
```

Comment:
this is not giving the entire path to node right? consider renaming to getPolicyNodePathPrefix